pkgs/by-name/in/intel-graphics-compiler/package.nix

```diff
@@ -26,8 +23,8 @@ let
     hash = "sha256-7coQegLcgIKiqnonZmgrKlw6FCB3ltSh6oMMvdopeQc=";
   };
 
-  inherit (llvmPackages_15) lld llvm;
-  inherit (if buildWithPatches then opencl-clang else llvmPackages_15) clang libclang;
+  inherit (opencl-clang.llvmPkgs) lld llvm;
+  inherit (opencl-clang) clang libclang;
```

Comment:
Does this actually require an LLVM 15 LLD? Since it was never using a patched LLD, I’m guessing it would work with the regular `llvmPackages.lld`. (Unless it’s consuming LLD as a library?)

Similarly, does it actually need a wrapped patched Clang, or just the patched `libclang` and *some* wrapped Clang (or even GCC?) to compile its own code with? I’m not sure we can commit to keeping `wrapCCWith` working for dropped Clangs – we have conditionals in there that can be cleaned up after the drops, although I don’t think any of them would be immediately relevant to the use case here.